### PR TITLE
Error message update

### PR DESCRIPTION
**Pull request by jgkogan**
When istio is built from the cloned github version, the command:
 istioctl manifest apply --set profile=demo 
returns:
Error: failed to apply manifests: Asset profiles/demo.yaml not found

This update fixes it so that the error message is instead

Error: failed to apply manifests: compiled in charts not found in this development build, use --charts with local charts instead or run make gen-charts

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: operator/cmd/mesh/manifest-apply.go [hunk 1]
```diff
@@ -164,6 +165,9 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 	}
 	_, iops, err := GenerateConfig(inFilenames, setOverlay, force, restConfig, l)
 	if err != nil {
+		if subErr := helm.CheckCompiledInCharts(); subErr != nil {
```
Comment (howardjohn) [inline operator/cmd/mesh/manifest-apply.go:168]: Will this fail even if `--charts` is provided?

Comment (jgkogan) [reply operator/cmd/mesh/manifest-apply.go:168]: No it won't if --charts $ISTIO/istio/manifests is included.

**Review by ostromart — APPROVED** [review]

**Comment by ostromart** [comment]
Looks like it's already fixed, closing this.

**Pull request closed without merging**